src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -37,59 +35,41 @@ export interface RegisterData {
   date_of_birth: string;
 }
 
-const CODE_OF_CONDUCT_QUESTIONS: CodeOfConductQuestion[] = [
-  {
-    name: "termsAndConditions",
-    text: "I agree to SparkleVerse's Terms and Conditions",
-    link: "https://sparklever.se/terms-and-conditions",
-  },
-  {
-    name: "commonDecency",
-    text:
-      "I will endeavor not to create indecent experiences or content, and understand my actions may be subject to review and possible disciplinary action",
-  },
-  {
-    name: "regionalBurn",
-    text:
-      "I understand this is an Australian Regional Burn guided by the Ten Principles of Burning Man",
-  },
-];
-
```

Comment:
As chris commented elsewhere, these hardcoded questions won't be required anymore